README.md

```diff
@@ -40,4 +40,7 @@ coding!
 - Lidia Prado: [GitHub](https://github.com/LidiaIleana) [LinkedIn](https://www.linkedin.com/in/lidiaprado/)
 - Dana Jeon: [GitHub](https://github.com/danajeon) [LinkedIn](https://www.linkedin.com/in/dana-jeon-dev/)
 - Alicia Guirand: [GitHub](https://github.com/aliciaguirand) [LinkedIn](https://www.linkedin.com/in/aliciaguirand)
-- Anna Bowen:  [GitHub](https://github.com/bowenanna) [LinkedIn](https://www.linkedin.com/in/realannabowen/)
+- Anna Bowen: [GitHub](https://github.com/bowenanna) [LinkedIn](https://www.linkedin.com/in/realannabowen/)
+- Jessica Beazer [GitHub](https://www.linkedin.com/in/jessicabeazer/
+  https://github.com/jessanagilepmp) [LinkedIn](https://www.linkedin.com/in/jessicabeazer/
+  https://github.com/jessanagilepmp)
```

Comment:
It appears that her GitHub link was repeated within her LinkedIn link?

Reply:
Hey I was able to delete the duplicate thanks for seeing that !